components/ArticlePage.jsx

```diff
@@ -20,10 +23,30 @@ export function ArticlePage() {
       });
   }, [article_id]);
 
+  useEffect(() => {
+    setCommentsLoading(true);
+    fetch(
+      `https://my-nc-news-zyxm.onrender.com/api/articles/${article_id}/comments`
+    )
```

Comment:
Rather than using fetch, see if you can refactor this to axios and extract to a separate api.js file 👍